docs/OIDC.md

```diff
@@ -0,0 +1,50 @@
+# Authenticating with OpenID Connect
+Follow the steps below to authenticate with [Open ID Connect](https://www.microsoft.com/security/business/security-101/what-is-openid-connect-oidc):
+
+1. [Create a Microsoft Entra application and service principal](https://learn.microsoft.com/azure/developer/github/connect-from-azure?tabs=azure-portal%2Cwindows#create-a-microsoft-entra-application-and-service-principal)
+1. [Add federated credentials](https://learn.microsoft.com/azure/developer/github/connect-from-azure?tabs=azure-portal%2Cwindows#add-federated-credentials)
+1. [Create GitHub secrets](https://learn.microsoft.com/azure/developer/github/connect-from-azure?tabs=azure-portal%2Cwindows#create-github-secrets)
+
+1. Assign the [Trusted Signing Certificate Profile Signer](https://learn.microsoft.com/azure/trusted-signing/concept-trusted-signing-resources-roles#supported-roles) role to your service principal.
+   - Open your Trusted Signing Account in the Azure portal.
+     - **Note:** You can assign the role from your Resource Group or Subscription if you have multiple Trusted Signing accounts.
+   - Navigate to the Access Control (IAM) tab.
+   - Click 'Add role assignment'.
+   - Select 'Trusted Signing Certificate Profile Signer'.
+   - Next.
+   - Assign access to your 'User, group, or service principal' or 'Managed identity'.
+   - Review + assign.
```

Comment:
Nit:

These steps need to be executed in order, so an ordered list is needed.

Code:
```markdown
1. Top-level item 1
   1. Sub-item 1
   2. Sub-item 2
   3. Sub-item 3
2. Top-level item 2
```

Output:
1. Top-level item 1
   1. Sub-item 1
   2. Sub-item 2
   3. Sub-item 3
2. Top-level item 2